packages/core/src/layout/HomepageTimer/HomepageTimer.tsx

```diff
@@ -14,59 +14,85 @@
  * limitations under the License.
  */
 
-import React, { FC } from 'react';
+import React from 'react';
 import { HeaderLabel } from '../HeaderLabel';
+import { ConfigApi, useApi, configApiRef } from '@backstage/core-api';
 
 const timeFormat = { hour: '2-digit', minute: '2-digit' };
-const utcOptions = { timeZone: 'UTC', ...timeFormat };
-const nycOptions = { timeZone: 'America/New_York', ...timeFormat };
-const tyoOptions = { timeZone: 'Asia/Tokyo', ...timeFormat };
-const stoOptions = { timeZone: 'Europe/Stockholm', ...timeFormat };
-
-const defaultTimes = {
-  timeNY: '',
-  timeUTC: '',
-  timeTYO: '',
-  timeSTO: '',
-};
 
-function getTimes() {
+class TimeObj {
```

Comment:
No need to do this in TS :grin: you can just define the following type and use object literals instead:

```ts
type TimeObj = {
  time: string;
  label: string;
}
```

Below you'd replace `new TimeObj(time, label)` with `{ time, label }`

Reply:
oh nice! Learning new things each time :) Updated.